### ISSUE TITLE
Add Skeleton as Bower package

### ISSUE THREAD

**Issue by dhg** [dhg/Skeleton#197]
Check out the info here and get 'er done: http://bower.io/docs/creating-packages/

**Comment by palmgren** [comment]
This would be really handy!  I actually was just on this project's page looking for its Bower package, so I'm glad to see it's coming soon.

**Comment by JakeHP** [comment]
This would be great.

**Comment by rizqyhi** [comment]
Just lookin for this too.
+1

**Comment by eabait** [comment]
:+1:

**Comment by JakeHP** [comment]
https://github.com/dhg/Skeleton/pull/200

**Comment by johngerome** [comment]
:+1:

**Comment by izolate** [comment]
dhg pls

**Comment by dhg** [comment]
Heyo! I promise I'm going to be dig into this shortly. I've been on a short holiday vacation, but will be back tomorrow! SOOOON.

**Comment by izolate** [comment]
enjoy the last of your holiday!

**Comment by dhg** [comment]
Alright, I'm starting to look into this and it all seems straightforward, but there are couple things I'm not sure what the best practice would be. Here are my quandaries:
1. I want to identify Normalize as a dependency. I'm guessing it's not best practice just to include my version of Normalize in a bundle, but rather a reference it as a dependency. I know I can do that, but is there anything special I should be considering outside of throwing the latest version of Normalize as a dependency of Skeleton?
2. Index.html is a pseudo-critical part of Skeleton. The meta tags, Raleway Google font, favicon, etc seem like they're valuable. Is that something that would usually be included in a Bower package or is it wiser to relegate that all to some form of documentation of the Bower package?

There's an existing PR that gets this started here: https://github.com/dhg/Skeleton/pull/200. 

I honestly haven't used Bower much, so any advice on the above would be greatly appreciated!

**Comment by izolate** [comment]
1. The best way would be to add `"normalize-css": "^3.0.2"` to `devDependencies` and use [gulpjs](http://gulpjs.com/) to concat both skeleton & normalize into a minified `skeleton.min.css`, to be placed in `dist/`. The alternative is to do this manually and include normalize inside the package.
2. Bower packages are mostly useful for their `.js` and `.css` components. Would recommend you add this to your `bower.json`:

```
{
  ...
  "ignore": [ "./index.html" ]
}
```

If you like, I can help out with a PR. Edit: https://github.com/dhg/Skeleton/pull/217

**Comment by dhg** [comment]
@izolate Would love additional help here. Seems like @nicolasmccurdy's work is a good start (https://github.com/dhg/Skeleton/pull/200). Any suggestions or PR's are welcome.

**Comment by dhg** [comment]
Alright - I finally did this. I don't think it's 100% properly executed, but at least it works for now. Running `bower install skeleton` or `bower install skeleton-css` will resolve in the latest version of the entire `master` branch being installed. Here's are a few details:
- Normalize & index.html are included which I understand might be non-standard, but makes sense to me for now until I can figure out a better way to handle it. Normalize is a dependency, but is packaged with Skeleton distro anway, so why not just keep that one. Index.html is an important file for font-linking, meta tags, etc...I think it should be included. 
- Not sure if I should add an ignore section to the `bower.json` and if I did, what exactly I would exclude, but I think for now leaving everything is alright in the short term

Thanks all. I'm going to close this for now, but would love for anyone to open PRs against what I have to improve the Bower experience. I'm not super familiar with FE package distribution - the most I use Bower for typically is just installing an occasional package. 

Thanks so much all and lemme know if I missed anything, but hopefully this is a step in the right direction.
